coalib/results/TextPosition.py

```diff
@@ -31,3 +31,27 @@ def line(self):
     @property
     def column(self):
         return self._column
+
+    def __le__(self, other):
+        if self.line is None or other.line is None:
+            return True  # if line is None, consider whole file
+        if self.line == other.line:
+            return (True
+                    if self.column is None or  # if column is None,
+                    other.column is None  # consider whole line
```

Comment:
when you document properly, I think you don't need those comments then :)
(especially as you don't use them below 😄 )